spec/lib/google_maps_geocoder_spec.rb

```diff
@@ -44,19 +44,19 @@
     it { should be_partial_match }
 
     context 'address' do
-      it { expect(subject.formatted_street_address).to eq '1600 Pennsylvania Avenue Northwest' }
+      it { expect(subject.formatted_street_address).to eq '1600 Pennsylvania Avenue Southeast' }
```

Comment:
haha, hope it's not! But seems like they actually changing this information. For example, now I have "Street" vs "St" issue in specs.

Not quite sure what could be done here, so just changed it again :confused: